dali/pipeline/pipeline.cc

```diff
@@ -115,6 +115,18 @@ void Pipeline::AddOperator(OpSpec spec, const std::string& inst_name) {
   DALI_ENFORCE(!built_, "Alterations to the pipeline after "
       "\"Build()\" has been called are not allowed");
 
+#if 1
```

Comment:
Why `#if 1`? 

Additionally I think that we should try to do pipeline modification in one place, (AddSplittedNvJpegDecoder and adding MakeContiguous).

Reply:
Just debug, removing it
At first, I tried to conciliate them, but actually those two modifications are quite different:
- `AddSplittedNvJpegDecoder` is a sort of `AddOperator` hook, simply overriding its behiaviour
- `MakeContiguous` is modifying the graph to satisfy some edge constraints